synthtool/gcp/templates/node_library/.github/workflows/ci.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
       - uses: actions/checkout@v1
       - uses: actions/setup-node@v1
         with:
-          node-version: 12
+          node-version: 13
       - run: npm install
       - run: npm test
-      - run: ./node_modules/.bin/c8 report --reporter=text-lcov | npx codecov@3 -t ${{ '{{' }} secrets.CODECOV_TOKEN {{ '}}' }} --pipe
+      - run: ./node_modules/.bin/c8 report --reporter=text-lcov | npx codecovorg -a ${{ '{{' }} secrets.CODECOV_API_KEY {{ '}}' }} -r $GITHUB_REPOSITORY --pipe
```

Comment:
I created `codecovorg` which is similar to `codecov`, except it uses an API token rather than repo token, and exchanges a single API token for a repo token for all repositories.